packages/docusaurus-plugin-content-blog/src/blogUtils.ts

```diff
@@ -27,7 +27,8 @@ import {
   isUnlisted,
   isDraft,
 } from '@docusaurus/utils';
-import {validateBlogPostFrontMatter} from './frontMatter';
+import {readLastUpdateData} from '@docusaurus/utils/lib/lastUpdateUtils';
```

Comment:
```suggestion
import {readLastUpdateData} from '@docusaurus/utils';
```

Shouldn't import from `pkg/lib`, it will be forbidden in the future